.circleci/config.yml

```diff
@@ -104,6 +104,7 @@ workflows:
             branches:
               only:
                 - develop
+                - release
```

Comment:
Need to handle dates in the release branch name

```suggestion
                - /^release-.*/
```

Reply:
(discussed on slack)